lib/launch/launch_web.dart

```diff
@@ -13,9 +24,54 @@ class WebLaunchType extends LaunchType {
   bool canLaunch(String path, LaunchData data) => path.endsWith('.html');
 
   Future<Launch> performLaunch(LaunchManager manager, LaunchConfiguration configuration) {
-    // TODO:
-    return new Future.error(new UnimplementedError());
+    Browser browser = deps[BrowserManager].browser;
+    if (browser == null) {
+      atom.notifications.addWarning('No browser configured.');
+      return new Future.value();
+    }
+
+    // Find pub serve for 'me'.
+    ServeLaunch pubServe = manager.launches.firstWhere((l) =>
```

Comment:
In that setup, the `serve` output would be just part of the launches output, and user's wouldn't have to know to start the `pub serve` before the web launch.